### ISSUE TITLE
Remove toolbar from png

### ISSUE THREAD

**Issue by jherreral** [holoviz/holoviews#3751]
When a plot is exported to png using `hv.save()` and Bokeh renderer the toolbar is plotted by default. This is also true if `hv.render()` and `bokeh.io.export_png()` are used. On the other side, when exporting to svg using `hv.render()` and `bokeh.io.export_svgs()` the toolbar is left out by default.
There are quick workarounds like `hv.save(myPlot.options(toolbar=None),'myExample.png')` that make the change inplace, but I think that it should be left out by default like in svg.

Using `hv.save(myPlot,'myDirtyPlot.png)`
_Why would a png have a toolbar?_
![myDirtyPlot](https://user-images.githubusercontent.com/33136201/58712571-d2fe8100-838e-11e9-86ff-3d504e391e22.png)

Using `hv.save(myPlot.options(toolbar=None),'myNeatPlot.png')`
_This is what I expected!_
![myNeatPlot](https://user-images.githubusercontent.com/33136201/58712561-cda13680-838e-11e9-8f05-3edc41115ceb.png)

**Comment by jbednar** [comment]
I agree; PNGs only need toolbars on the Bokeh site, for docs about Bokeh; otherwise I think they should be gone by default.
